### ISSUE TITLE
auto import not working with typescript

### ISSUE THREAD

**Issue by psi-4ward** [quasarframework/vue-cli-plugin-quasar#32]
**Describe the bug**

I've a setup with typescript and quasar but the bundle size analyzing shows 716kb from `quasar/src/index.esm.js`

**To Reproduce**

1.) setup a vue project with vue-cli
2.) `vue add quasar`
3.) compile and analyze

**Expected behavior**
Small bundle size :)

babel.config.js:

```js
module.exports = {
  "presets": [
    "@vue/cli-plugin-babel/preset"
  ],
  "plugins": [
    [
      "transform-imports",
      {
        "quasar": {
          "transform": "quasar/dist/babel-transforms/imports.js",
          "preventFullImport": true
        }
      }
    ]
  ]
}
```

vue.config.js
```
module.exports = {
  pluginOptions: {
    quasar: {
      importStrategy: 'kebab',
      rtlSupport: false,
    }
  },
  transpileDependencies: [
    'quasar'
  ]
}
```

tsconfig.json
```
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "strict": true,
    "jsx": "preserve",
    "importHelpers": true,
    "moduleResolution": "node",
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "sourceMap": true,
    "baseUrl": ".",
    "types": [
      "webpack-env"
    ],
    "paths": {
      "@/*": [
        "src/*"
      ]
    },
    "lib": [
      "esnext",
      "dom",
      "dom.iterable",
      "scripthost"
    ]
  },
  "include": [
    "src/**/*.ts",
    "src/**/*.tsx",
    "src/**/*.vue",
    "tests/**/*.ts",
    "tests/**/*.tsx"
  ],
  "exclude": [
    "node_modules"
  ]
}
```

**Comment by rstoenescu** [comment]
Do you also have babel in your project? When you created the vue-cli project?

**Comment by psi-4ward** [comment]
Yes I think so. I put some debug log into imports.js and saw that `Dialog` and some other Plugins went through the transformer but none of Quasar-Components.

**Comment by psi-4ward** [comment]
this still exists - any ideas?

**Comment by rstoenescu** [comment]
Fix will be available today in "vue-cli-plugin-quasar" v3.0.0.
Please read the upgrade guide (after released).
